main.js

```diff
@@ -148,7 +148,7 @@ module.exports = function(options) {
 				directory + '/views/partials'
 			],
 			defaultLayout: false,
-			layoutsDir: options.layoutsDir || __dirname + '/layouts',
+			layoutsDir: options.layoutsDir || process.cwd() + '/client/layouts',
```

Comment:
shouldn't this point to something like `process.cwd() + '/bower_components/n-layout/layouts'`